### ISSUE TITLE
AddToCart event on Ajax Add To Cart pop-up not triggering

### ISSUE THREAD

**Issue by BitcoinKid9** [tensworth/Shopify-Facebook-Pixel#3]
Hi, your code looks great. Thank you!
If your Shopify theme uses an Ajax pop-up for AddToCart the pixel is not firing on that event. Everything else is working great except for fbq('track', 'AddToCart'); event.
Please let me know if you have any answer for getting this event to trigger on Ajax pop-up AddToCart?
Thanks for any advice...Regards, James

**Comment by tensworth** [comment]
You are right. This will not capture any events that do not require full page reload. Any AJAX events will not be captured. Your best bet is to use Google tag manager to capture the click of the add to cart and then fire the pixel. 

I may look in to solving this using JS in the future if there seems to be enough demand.
